common/src/stack/report-system/command/report/system/tests/test_sles_patch.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+import os
+import stack.api
+import hashlib
+import json
+
+
+# Returns hash of filename
+def get_hash(filename):
+	hasher = hashlib.md5()
+	with open(filename, 'rb') as f:
+		hasher.update(f.read())
+
+	return hasher.hexdigest()
+
+
+# Verify SLES patches are installed
+def test_sles_pallet_patched(host, report_output):
+	total_matched = []
+	base_dir = '/export/stack/pallets/SLES/'
+	sles_flavors = os.listdir(base_dir)
+	if not sles_flavors:
```

Comment:
If you move this up a line and change it to checking if `base_dir` exists, that should be enough to make the test past on centos.